youtube_dl/extractor/tele5.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from .nexx import NexxIE
+
+
+class Tele5IE(InfoExtractor):
+    _VALID_URL = r'https://www.tele5.de/(mediathek/filme-online/videos|tv/).*'
```

Comment:
1. `.*` at the end is senseless.
2. Don't capture groups you don't use.
3. Capture and use display id.